docs/protocol/src/dex/action/swap.md

```diff
@@ -12,12 +12,28 @@ The invariants that the Swap upholds are described below.
 
 2. The swap binds to a specific claim address.
 
+3. The swap does not reveal the swapper identity. The swap *does* reveal the assets being swapped, as well as the amounts of those assets.
+
+    3.1 The swap data included in a transaction preserves this property.
+
+    3.2 The integrity checks as described above in invariant 1 are done privately.
+
+4. The balance contribution of the pre-paid claim fee is private.
```

Comment:
```suggestion
4. The swap does not reveal the pre-paid claim fee.
```
I think this would make it clearer that the claim fee is what's intended to be private. WDYT?